package.json

```diff
@@ -36,7 +36,7 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "^3.4.1",
     "typescript": "^4.0.2",
-    "uw-content-validation": "^0.8.9"
+    "uw-content-validation": "^0.8.10"
```

Comment:
need to update to latest version.

Reply:
This is updated in the other/next pull request. So if both are done, then it is updated to 0.8.11.